### PR TITLE
rrd excluded to separate File

### PR DESCRIPTION
**Pull request by fbilly**
created new File rrdexport.py;
glucometer.py modified to use new file.

### CODE REVIEW

File: glucometer.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 from glucometerutils import common
 from glucometerutils import exceptions
+from glucometerutils import rrdexport
```
Comment (Flameeyes) [inline glucometer.py]: You should use a conditional import here, otherwise the script will only work when rrd is installed still.

File: glucometer.py [hunk 2]
```diff
@@ -61,6 +62,17 @@ def main():
...
+    'rrd', help='Dump values into RRD Database.'
+                'Please use same Values like the dump command (--driver and --device)')
+  parser_rrd.add_argument(
+    '--unit', action='store', choices=common.VALID_UNITS,
+    help='Select the unit to use for the dumped data.')
+  parser_rrd.add_argument(
+    '--sort-by', action='store', default='timestamp',
+    choices=common.Reading._fields,
+    help='Field to order the dumped data by.')
+
```
Comment (Flameeyes) [inline glucometer.py]: And this action should also be conditional to the import having completed.

File: glucometer.py [hunk 3]
```diff
@@ -104,6 +116,7 @@ def main():
           readings, key=lambda reading: getattr(reading, args.sort_by))
 
       for reading in readings:
+        #print(readings)
```
Comment (Flameeyes) [inline glucometer.py]: Unrelated change

File: glucometerutils/README.rrdexport [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Right now i have it only tested with glucometers I own.
+I connot guarranty that the graphs are correct for every glucometer.
+
+## Usage / Prerequisites
+
+as the primary Prerequisites you need to install / enable the python 3
+and the rrd module.
+
+for OpenSuSE:
+zypper in python-pyrrd python-rrdtool python-setuptools rrdtool rrdtool-devel python3-devel
```
Comment (Flameeyes) [inline glucometerutils/README.rrdexport]: Probably better to keep it in block code (triple-backtick): ```

File: glucometerutils/README.rrdexport [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Development
+
+If you want to contribute code, please note that the target language
+is Python 3.2, and that the style to follow is for the most part PEP8
+compatible.
+
+## License
+
+Copyright © 2017 Florian Ballangó <florianballango@gmx.de>
```
Comment (Flameeyes) [inline glucometerutils/README.rrdexport]: Just add your copyright note to the main README and avoid the duplication of license/development parts.

File: glucometerutils/rrdexport.py [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+
+import rrdtool
+import os.path
+import os
+#import subprocess
+import webbrowser
+
+
+from os.path import expanduser
```
Comment (Flameeyes) [inline glucometerutils/rrdexport.py]: You should really just write the output to the current work directory, as the home might not be the right place to output it at all.

File: glucometerutils/rrdexport.py [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+  "LINE2:180#ff8080:Critical\\r", "LINE2:80#8080ff:Warning \\r",
+  "LINE3:x3#FF0000", "LINE3:x2#000000", "LINE3:x1#ff951b",
+  #"LINE1:mmol#ff0000:mmol/l", "--right-axis-label", "mmol/l",
+  "GPRINT:First:von %d.%m.%Y %H\:%M   --:strftime", "GPRINT:Last:bis %d.%m.%Y %H\:%M\\c:strftime",
+  "--color", "BACK#FFFFFF", "--color", "CANVAS#e5e5e5", "--color", "SHADEB#9999CC"
+  )
+  print ("the Graph is created: " + rrdgraphic )
+  #os.system("gwenview " + rrdgraphic)
+  webbrowser.open(rrdgraphic)
+  return 0
```
Comment (Flameeyes) [inline glucometerutils/rrdexport.py]: No need to return here.

File: glucometerutils/rrdexport.py (hunk 6 above)
Comment (fbilly) [reply glucometerutils/rrdexport.py]: I have used the home directory, because you could use this python for multiple users. So you have a database and graphs for every user in the home directory.
Wich folder do you prefer ?

File: glucometer.py (hunk 2 above)
Comment (fbilly) [reply glucometer.py]: Hi, i have a little problem how to do this. How can i do the import and the arguments conditional at the same time ?

**Comment by fbilly** [comment]
Hi Diego, 
i have updated the files with your requested changes in my repo.
Do i need to create a new pull request or do you see the changes ?

Best regards,
Florian
